scoala-online/main
# Feat/#84 create an express app

### This pull request solves issue:

Closes: #84

### Which app is this issue related to?

- [x] Admin Panel

### Types of changes


### Wiki Updates


### Testing checklist


#### Security Filters - **_For API Only_**

**_Please make sure to add the respective Security Filters before submitting a PR_**


#### Compatibility tests - **_For E-Learning React App Only_**


### What changes have been done within this issue?

An Express app has been created with a home page featuring a list of resources that can be managed.
Each resource has its own page where it displays all entries received from the API.

### How should this be manually tested?

Pre-requisite: The API needs to be running.
Serve the app with: `nx serve admin-panel` and visit the home and endpoint pages to make sure that they work and display the correct data.

### Screenshots or example usage
